Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿Console.WriteLine("введите первое число");
+int m = int.Parse(Console.ReadLine());
+Console.WriteLine("введите второе число");
+int n = int.Parse(Console.ReadLine());
+int s = (n+m)%2;
+static bool sravnenie(ref int s,ref int m,ref int n)
+{
+    if (s == 0 & (m*n)%3 == 0)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+Console.WriteLine(sravnenie(ref s, ref m, ref n));
+
+Console.WriteLine("введите х");
+double x = double.Parse(Console.ReadLine());
+Console.WriteLine("введите y");
+double y = double.Parse(Console.ReadLine());
+static bool koordinati(ref double x, ref double y)
+{
+    if((x>= 2 & y>=0)|(x>=1 & y<=-1))
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+Console.WriteLine(koordinati(ref x, ref y));
+
+Console.WriteLine("Введите позицию белого коня");
+var whitehorsepositin = Console.ReadLine();
+
+static void decodeposition(string position, out int row, out int colomn)

```

Comment:
Используйте для имен методов стиль PascalCase.
```suggestion
static void DecodePosition(string position, out int row, out int colomn)

```